samples/routing/custom/Core_6/Billing/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.CustomRouting.Billing";
```

Comment:
True, but I noticed all samples I've seen don't have that as a constant. Should we do this accross the board? Or maybe use the convention e.g. (`path.to.sample.solution.project_name`)